flink-formats/flink-parquet/src/test/java/org/apache/flink/formats/parquet/protobuf/ParquetProtoStreamingFileSinkITCase.java

```diff
@@ -49,13 +50,12 @@
  * Simple integration test case for writing bulk encoded files with the {@link StreamingFileSink}
  * with Parquet.
  */
-public class ParquetProtoStreamingFileSinkITCase extends AbstractTestBase {
-
-    @Rule public final Timeout timeoutPerTest = Timeout.seconds(20);
+@ExtendWith(MiniClusterExtension.class)
```

Comment:
Is there a difference between using `@ExtendWith` and `@RegisterExtension` on a field member like you did it in `AvroParquetFileReadITCase`? :thinking: shall we go for a consistent approach if not?